setup.cfg

```diff
@@ -7,6 +7,7 @@ license_files =
     commoncode.ABOUT
     README.rst
 name = commoncode
+version = 31.0.0
```

Comment:
If you use setuptools scm, this should not be set AFAICS

Reply:
@pombredanne I removed this, and now when I try to build a wheel for the the 31.0.1 release of commoncode, the version on the wheel is 0.0.0.